plugins/modules/wafv2_web_acl.py

```diff
@@ -87,6 +88,13 @@
           description:
             - Rule configuration.
           type: dict
+    custom_response_bodies:
+      description:
+        - A map of custom response keys and content bodies. Define response bodies here and reference them in the rules by providing
+        - the key of the body dictionary element.
+        - Each element must have a unique dict key and in the dict two keys for I(content_type) and I(content). Needs botocore >= 1.21.0 to work.
+      type: dict
+      version_added: 3.0.0
```

Comment:
```suggestion
      version_added: 3.1.0
```

We've missed the train for 3.0.0

Reply:
fixed